src/test/java/teammates/test/pageobjects/InstructorStudentRecordsPage.java

```diff
@@ -80,11 +81,25 @@ public boolean verifyEditCommentButtonClick(int id) {
                        && !getCommentEditLink(id).isDisplayed();
         return display;
     }
+    
+    public void clickEditCommentAndCancel(int id) {
+        getCommentEditLink(id).click();
+        getCommentEditCancelLink(id).click();
+    }
+    
+    public void verifyCommentEditBoxNotVisible(int id) {
+        assertFalse(isElementVisible(By.id("commentTextEdit" + id)));
+    }
 
     private WebElement getCommentEditLink(int id) {
         return browser.driver.findElement(By.id("commentedit-" + id));
     }
 
+    private WebElement getCommentEditCancelLink(int id) {
+        return browser.driver.findElement(By.id("commentsave-" + id))
+                             .findElement(By.xpath("./following-sibling::input"));
```

Comment:
This is an assumption that the Cancel button must be next to the Save button. At the moment, the Cancel button doesn't have an ID so I recommend to give it an ID later if possible. For now, it is good enough for a quick fix.